### ISSUE TITLE
Allow specifying separate appicon vertical and horizontal padding

### ISSUE THREAD

**Issue by ndias** [home-sweet-gnome/dash-to-panel#1330]
I'm Fedora34 with gnome 40 and using the latest dev from github
Screenshots from before and after
![Screenshot from 2021-05-10 12-31-33](https://user-images.githubusercontent.com/886782/117654435-a7e15680-b18d-11eb-84c1-3772da7e25e7.png)
![Screenshot from 2021-05-10 12-31-06](https://user-images.githubusercontent.com/886782/117654427-a57efc80-b18d-11eb-836d-bddc9a14fcfe.png)

**Comment by LexRiver** [comment]
Yes, please make vertical padding and horizontal padding independent in settings.
For example, I want vertical padding to be greater then horizontal padding to make an icons small but not to distant between each other in horizontal direction.
